.github/workflows/backport-pull-request.yml

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
missing license header. Since it contains copied code from documentation also its header might be necessary to be added

Reply:
there is no reason to not start here. Especially since it contains code inspired by example configuration